ts/components/conversation/header/ConversationHeader.tsx

```diff
@@ -57,3 +72,71 @@ export const ConversationHeaderWithDetails = () => {
     </div>
   );
 };
+
+const RecreateGroupContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  align-self: center;
+  width: 100%;
+
+  .session-button {
+    padding-inline: var(--margins-3xl);
+  }
+`;
+
+function useShowRecreateModal() {
```

Comment:
Moving this outside the component in this case seems like overkill.

Reply:
but I like it like this :)